packages/numactl/Makefile

```diff
@@ -32,13 +32,13 @@ define Package/numactl
    CATEGORY:=Utilities
    TITLE:=Control NUMA policy for processes or shared memory
    URL:=https://github.com/numactl/numactl
-   DEPENDS:=@TARGET_x86_64 +libpthread +libstdcpp +libc +librt +libnuma
+   DEPENDS:=@(x86_64||aarch64) +libpthread +libstdcpp +libc +librt +libnuma
```

Comment:
Don't know if this should include `arm` as well.  It's not clear from the `numactl` `README.md` that 32-bit processors are adequately supported.

Reply:
I will try it on x86 and arm but in 2 weeks.

Reply:
ARM is obsolete. aarch64 is OK.